data/mods/gen7mixandmega/scripts.ts

```diff
@@ -99,10 +99,12 @@ export const Scripts: ModdedBattleScriptsData = {
 				species.types = [species.types[0], deltas.type];
 			}
 			const baseStats = species.baseStats;
-			for (const statName in baseStats) {
+			let statName: StatID;
+			for (statName in baseStats) {
 				baseStats[statName] = this.battle.clampIntRange(baseStats[statName] + deltas.baseStats[statName], 1, 255);
 			}
 			species.weighthg = Math.max(1, species.weighthg + deltas.weighthg);
+			// @ts-ignore
```

Comment:
ditto

Reply:
This at the least needs an explanation if it's not going to be converted to a cast.